### DISCUSSION TITLE
Audio mute through special function

### DISCUSSION THREAD

**Discussion by onki69** [EdgeTX/edgetx#1041] [Ideas]
Hi,

The Radiomaster TX16s is missing a phone jack as well as wireless audio output.
To get wireless audio through BT several tweaks by using a BT audio module connected to the Audio_out of the processor are possible.

There also is a Audio_mute signal coming from the processor in order to mute the amplifier (but still have audio coming from the µC.

It would be great if this mute could be controlled through a switch managed by the special functions.

In addition a control of Aux1 and/or Aux2 supply (throiugh special functions) would be appreciated in order to switch the BT-audio module on and off.

All this could be done by using dedicated hardware switches but I hate to drill additional holes in my transmitter:-).

Best regards
Onki

**Comment by onki69** [comment]
I placed this in issues because it seems more like a feature request. Can be deleted here.

Best regards
Onki

**Comment by brainbubblersbest** [comment]
On x12s and fros. R9 doesnt trigger the well known mobile Phone noise in the speakers. It's already discovered that muting or shutdown the amplifier can solve that. 
It seems that this is done by fros. But it's not utilized by otx neither edgetx. 

Shutdown or muting the amp when no Sound is played, then reenabling it with some delay, can maybe delete that disturbing Sounds when flying with 900mhz Hardware. 

Dont know if this would work on other Radios like tx16s also. 
But the JT16 missuse that shutdown pin from the amp as a Mute function. 

https://github.com/opentx/opentx/issues/8054#issue-727664262
